docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -122,8 +122,8 @@ blocks every `blockperiodseconds`.
 
     An internal PegaSys IBFT 2.0 testnet has four geographically dispersed validators in Sweden,
     Sydney, and two in North Virginia. With a `blockperiodseconds`of 5 and a
-    `requesttimeoutseconds` of 10, the testnet consistently creates blocks with a five second
-    blocktime.
+    `requesttimeoutseconds` of 10 (`blockperiodseconds` x 2), the testnet consistently creates
```

Comment:
1. The network is not longer running, so may need to change
2. Where is our guidance of reqeusttimeout=2xblocktime? Its not a _bad_ suggestion,  but its probably better to suggest that the numbers should be tuned to meet your network performance, and required block propogation rate.

Reply:
I updated with suggestion to requesttimeout = block time + (some network communication x number of nodes)

Reply:
Updated per Trent's feedback